cfgen/cfgen

```diff
@@ -1092,7 +1092,10 @@ def pool_drives(m0conf: Dict[Oid, Any],
                        for drive_id, node_id in all_drives
                        if ref['path'] == m0conf[m0conf[drive_id].sdev].filename
                        and ref.get('node') in (None, m0conf[node_id].name)]
-            assert len(targets) == 1  # guaranteed by validate_pools_desc()
+            assert len(targets) == 1, \
+                '{} {}'.format(ref['path'], 'has wrong node: {}'.format(
```

Comment:
For me this code looks too complicated. `ref['path']` could have been refactored into a variable to improve readability, probably there is also a way to avoid ternary operator..